src/Components/Dialogs/dialogs.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 
-import I18Y from '../../Core/constants';
-import GetItem from './GetItems/getItem';
+import { I18Y, dialogsData } from '../../core/constants';
```

Comment:
Separate the groups of related imports with a new line.

Reply:
Yes.It's really look like bad